app/services/appointment_notification_service.rb

```diff
@@ -16,6 +16,7 @@ def initialize(appointments:, days_overdue: 3)
 
   def send_after_missed_visit
     eligible_appointments = appointments.eligible_for_reminders(days_overdue: days_overdue)
+    # Exclude patients enrolled in an experiment
 
```

Comment:
We'll need to be able to figure out whether an appointment is part of an experiment or not, so we can skip sending the usual 3 days overdue message. We could do this through a method that calculates based on enrollment and dates, or alternatively, add a nullable `experiment_id` column on appointments.